packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -108,6 +108,10 @@ sealed class _DebugSemanticsRoleChecks {
     SemanticsRole.tab => _semanticsTab,
     SemanticsRole.tabBar => _semanticsTabBar,
     SemanticsRole.tabPanel => _noCheckRequired,
+    SemanticsRole.table => _noCheckRequired,
+    SemanticsRole.cell => _semanticsCell,
+    SemanticsRole.row => _noCheckRequired,
```

Comment:
shouldn't this check if they are under table as well?

Reply:
I didn't add a role for table row in this PR so i didn't add row checks here

Reply:
is the engine side properly implemented? if not this should just throw if someone attempt to use it

Reply:
I will throw a unimplemented error for row here